packages/router/router.ts

```diff
@@ -2985,6 +3010,18 @@ function isQueryRouteResponse(obj: any): obj is QueryRouteResponse {
   );
 }
 
+function isValidMethod(method: string | undefined): method is FormMethod {
+  if (!method) return false;
+  return validRequestMethods.has(method as FormMethod);
```

Comment:
Personal thing, but it seems a bit weird to do an early return if the main functionality is only a oneliner

```suggestion
  return method
    ? validRequestMethods.has(method as FormMethod)
    : false;
```

Reply:
I ended up removing the `undefined` case for these utils since it was making them a bit ambiguous